spark_log_parser/loaders/json.py

```diff
@@ -13,53 +14,72 @@
 logger = logging.getLogger("JSONLoaders")
 
 
-class JSONBlobDataLoader(DataLoader, Generic[RawJSONBlobDataLoader]):
+class JSONBlobDataLoader(DataLoader):
     cache = False
     blob_data_loader: RawJSONBlobDataLoader
 
     def __init__(self, blob_data_loader: RawJSONBlobDataLoader, **kwargs):
         super().__init__(**kwargs)
         self.blob_data_loader = blob_data_loader
 
+    @staticmethod
+    def _yield_json(data):
+        _, blob = data
+        return orjson.loads(next(blob))
```

Comment:
Hmm, true. Perhaps it would be better to rename it to something like `_parse_file()`. I didn't want to force callers to iterate over the result since it'll always be 1 value.

Reply:
I was thinking it could be used in `batch_log_fn` like,
```
[data for data in self._yield_parsed(raw_datas)]
```

Reply:
oh yeah that's pretty clean as well!